lib/babe/verify_integration_test.go

```diff
@@ -284,28 +256,35 @@ func TestVerificationManager_VerifyBlock_MultipleEpochs(t *testing.T) {
 	require.NoError(t, err)
 
 	// create block in future epoch
-	block1 := createTestBlock(t, babeService, genesisHeader, [][]byte{},
-		cfg.EpochLength*futureEpoch+1, futureEpoch, futureEpochData)
-	block2 := createTestBlock(t, babeService, &block1.Header, [][]byte{},
-		cfg.EpochLength*futureEpoch+2, futureEpoch, futureEpochData)
+	slot1 := getSlot(t, rt, time.Now())
+	//slot1.number = babeCfg.EpochLength*futureEpoch + 1
+	block1 := createTestBlockWithSlot(t, babeService, &genesisHeader, [][]byte{}, futureEpoch, futureEpochData, slot1)
+
+	slot2 := getSlot(t, rt, time.Now())
+	//slot2.number = babeCfg.EpochLength*futureEpoch + 2
+	block2 := createTestBlockWithSlot(t, babeService, &block1.Header, [][]byte{}, futureEpoch, futureEpochData, slot2)
```

Comment:
Remove commented code?

Reply:
I actually kept this one intentionally to use as reference in my follow up PR